apps/frontend/src/pages/auth/components/sign-up-form/libs/constants.ts

```diff
@@ -1,6 +1,7 @@
 import { type UserSignUpRequestDto } from "~/modules/users/users.js";
 
 const DEFAULT_SIGN_UP_PAYLOAD: UserSignUpRequestDto = {
+	confirmPassword: "",
```

Comment:
Do we need to send confirmPassword to server? I feel like this validation can be done on client

Reply:
Yeah this field is not needed on server. But in useAppForm hook specified type UserSignUpRequestDto, so I can't add field that is missing in this type. And when I delete property confirmPassword from UserSignUpRequestDto I faced an error with confirm password input.